__tests__/app.test.js

```diff
@@ -193,3 +193,55 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("should return status code 200 and respond with updated article object", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 3 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 103,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+
+  test("should return 400 Bad Request if bad article_id", () => {
+    return request(app)
+      .patch("/api/articles/uwotm8")
+      .send({ inc_votes: 3 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+
+  test("should return 400 bad request if article_id doesn't exist", () => {
```

Comment:
Update this description to 404 not found

Reply:
Oops - thanks!